src/app/components/mnoe-config/mnoe-config.svc.coffee

```diff
@@ -100,6 +100,33 @@ angular.module 'mnoEnterpriseAngular'
         $log.debug("DASHBOARD_CONFIG.provisioning.enabled missing")
         false
 
+    @arePublicPagesEnabled = () ->
+      if DASHBOARD_CONFIG.public_pages?.enabled?
+        DASHBOARD_CONFIG.public_pages.enabled
+      else
+        $log.debug("DASHBOARD_CONFIG.public_pages.enabled missing")
+        false
+
+    @isPublicPricingEnabled = () ->
+      if DASHBOARD_CONFIG.public_pages?.display_pricing?
+        DASHBOARD_CONFIG.public_pages.display_pricing
+      else
+        $log.debug("DASHBOARD_CONFIG.public_pricing.display_pricing missing")
+
+    @publicApplications = () ->
```

Comment:
Maybe rename this to `@arePublicApplicationsEnabled`

Reply:
I renamed the function above `@arePublicPagesEnabled` to `@arePublicApplicationsEnabled`. The function, `@publicApplications` returns an array of app nids